CHANGELOG.md

```diff
@@ -3,6 +3,30 @@ Changes by Version
 
 <!-- next version -->
 
+## 0.2.0
```

Comment:
I went with `0.2.0` instead of `0.1.1` because of the deployment rename breaking change